app/lib/intro_screen.dart

```diff
@@ -77,6 +80,90 @@ class IntroScreen extends StatelessWidget {
   }
 }
 
+class _ClientRecords extends StatefulWidget {
+  const _ClientRecords();
+
+  @override
+  State<_ClientRecords> createState() => _ClientRecordsState();
+}
+
+class _ClientRecordsState extends State<_ClientRecords> {
+  Future<List<UiClientRecord>>? _clientRecords;
+
+  @override
+  void initState() {
+    super.initState();
+    loadClientRecords();
+  }
+
+  void loadClientRecords() async {
+    final clientRecords = User.loadClientRecords(dbPath: await dbPath());
+    setState(() {
+      _clientRecords = clientRecords;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder<List<UiClientRecord>>(
+      future: _clientRecords,
+      builder: (context, snapshot) {
+        if (snapshot.hasData) {
+          return _ClientRecordsList(snapshot.data!);
+        } else if (snapshot.hasError) {
+          return Text(
+            'Error loading contacts',
+          );
+        }
+        return const CircularProgressIndicator();
+      },
+    );
+  }
+}
+
+class _ClientRecordsList extends StatelessWidget {
+  const _ClientRecordsList(this.clientRecords);
+
+  final List<UiClientRecord> clientRecords;
+
+  @override
+  Widget build(BuildContext context) {
+    const itemExtent = 72.0;
+
+    return Center(
+      child: ConstrainedBox(
+        constraints: BoxConstraints(
+          maxHeight: 3.5 * itemExtent, // 3 items without scrolling
+          maxWidth: MediaQuery.of(context).size.width.clamp(0, 400),
+        ),
```

Comment:
I know this is a developer feature only, but if we prevent scrolling we might still miss something. Maybe add an indication if there are more than 3 clients or something.

Reply:
This is actually a valid concern. Also, it seems like an anti-pattern. ListViews are usually not constrained in height, so this layout creates a strange iFrame-like feeling.

I think it would be better to remove user selection from the Intro screen entirely and continue logging in with the default user as before. Instead, we can add the user selection dialog in the developer settings and allow switching users there.

The only concern I have is that the default user is always the last one created. So, it might make sense to add a field to the database to mark the last logged-in user as the default.

Reply:
That sounds sensible, switching users should be fast enough. It's fine to extend the DB as well. Just for context, down the road we are going to revise how DB encryption works and it might be that this developer feature then needs to go because of the revamp.

Reply:
After private communication we came with the following design and layout for the dev settings:

<img src="https://github.com/user-attachments/assets/24ac22e6-3d36-4240-89d7-c405e6a4d69a" width="400">